Finished loading dataset.
Finished loading dataset.
Finished loading dataset.
Data loaded. Start training...
/zhome/f6/e/127187/miniconda3/envs/py39/lib/python3.9/site-packages/torch/optim/lr_scheduler.py:247: UserWarning: To get the last learning rate computed by the scheduler, please use `get_last_lr()`.
  warnings.warn("To get the last learning rate computed by the scheduler, "
lr [0.010000000000000002, 0.1, 0.1, 0.1]
Epoch: [0/100], Regression loss: 0.15391144156455994, Transfer loss: 0.7308250665664673, Total loss: 0.8847365379333496, Domain acc.: 54.166664123535156

torch.Size([12, 3, 224, 224])
tensor([1.0000, 0.8273, 0.9842, 0.7977, 0.9623, 0.8400, 0.6982, 0.9077, 0.6275,
        0.8927, 0.9868, 0.9831], device='cuda:0')
[1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
Traceback (most recent call last):
  File "/work3/dgro/OrtSurDomainAdapt/main.py", line 164, in <module>
    main(args)
  File "/work3/dgro/OrtSurDomainAdapt/main.py", line 100, in main
    (val_loss, val_acc) = validate(val_loader, regressor, dann)
  File "/work3/dgro/OrtSurDomainAdapt/utils/analysis/validate.py", line 30, in validate
    y, f = model(x)
NameError: name 'x' is not defined
